doc/tables/code-managers.csv

```diff
@@ -0,0 +1,20 @@
+Affiliation;Code Manager;Areas of Expertise
+EPIC;**Michael Lueken (@MichaelLueken)**;Lead SRW App code manager
+GSL;Daniel Abdi (@danielabdi-noaa);Workflow generation, testing RRFS on the cloud, environment modules  
+GSL;Jeff Beck (@JeffBeck-NOAA);SRW App configuration/workflow, code management, meteorological evaluation  
+EMC;Ben Blake (@BenjaminBlake-NOAA);Output visualization, Rocoto  
```

Comment:
@gspetro-NOAA - We should add Brian Curtis (BrianCurtis-NOAA), from EMC, who will now be in charge of the Operational air quality modeling (Online-CMAQ).

Reply:
Thanks, @gspetro-NOAA!